Source/DafnyTestGeneration/GenerateTestsCommand.cs

```diff
@@ -49,9 +49,9 @@ private enum Mode {
   }
 
   private readonly Argument<Mode> modeArgument = new("mode", @"
-block - Prints block-coverage tests for the given program.
-branch - Prints branch-coverage tests for the given program.
-path - Prints path-coverage tests for the given program.");
+Block - Prints block-coverage tests for the given program.
+Branch - Prints call-graph-coverage tests for the given program.
```

Comment:
Is this text right, now that you've renamed the constructor?

Reply:
Oh, right, thank you for catching this. Have fixed it in my latest commit.